docker/rucio.cfg.j2

```diff
@@ -52,3 +55,10 @@ usercert = {{ RUCIO_CFG_WEBUI_USERCERT | default('/opt/rucio/etc/usercert_with_k
 [oidc]
 idpsecrets = {{ RUCIO_CFG_OIDC_IDPSECRETS | default('/opt/rucio/etc/idpsecrets.json') }}
 {% if RUCIO_CFG_OIDC_ADMIN_ISSUER is defined %}admin_issuer = {{ RUCIO_CFG_OIDC_ADMIN_ISSUER }}{% endif %}
+
+[client]
+rucio_host = {{ RUCIO_HOST | default('') }}
```

Comment:
Have these values been already defined in `values.yaml`? 
Either in the chart defatul `values.yaml` file or from  kustomize files?

Reply:
All variables are configured with the default value of `""` in `values.yaml`:
https://github.com/dmwm/CMSRucio/blob/9ef46f0ab9fa15ff51d1a2a4b4cf387b9f229d26/helm/rucio-consistency/values.yaml#L19-L27

And there are configured in `flux` using `kustomize` for the desired cluster (prod in this case):
https://github.com/dmwm/rucio-flux/blob/90c674e655f70c5f88bf1c0664e7d23c7c874847/apps/production/prod-consistency.yaml#L4-L11